<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/app.ico</normaloff>resources/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>811</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pagePapers">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>301</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Title</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Authors</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Keywords</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Journal Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>URL</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>DOI</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>251</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Tags</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>350</y>
         <width>90</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="generateBibliographyButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>530</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Bibliography</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshPdfButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPaperButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Paper</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showPapersButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Papers</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>341</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>671</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="setPdfFolderButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Set PDF Folder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bibliography</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>82</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Plain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IEEE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>APA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Springer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chicago</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MLA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BibTeX</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="bibliographyTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>611</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="copyButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>720</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>720</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="latexButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LaTeX</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regenerateButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regenerate</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
